zqd/handlers_zeek_test.go

```diff
@@ -92,30 +103,39 @@ func TestPacketPostZeekFailImmediate(t *testing.T) {
 }
 
 func TestPacketPostZeekFailAfterWrite(t *testing.T) {
-	exec := abspath(t, filepath.Join("testdata", "zeekwritefail.sh"))
-	p := packetPost(t, exec, "./testdata/valid.pcap", 202)
-	defer p.cleanup()
-	t.Run("TaskEndError", func(t *testing.T) {
-		expected := &api.TaskEnd{
-			Type:   "TaskEnd",
-			TaskID: 1,
-			Error: &api.Error{
-				Type:    "Error",
-				Message: "zeek exited with status 1: exit after writing logs",
-			},
-		}
-		last := p.payloads[len(p.payloads)-1]
-		require.Equal(t, expected, last)
-	})
-	t.Run("EmptySpaceInfo", func(t *testing.T) {
-		u := fmt.Sprintf("http://localhost:9867/space/%s", p.space)
-		var info api.SpaceInfo
-		httpJSONSuccess(t, zqd.NewHandler(p.core), "GET", u, nil, &info)
-		expected := api.SpaceInfo{
-			Name: p.space,
-		}
-		require.Equal(t, expected, info)
-	})
+	var tests = []struct{ name, script string }{
+		{"Fail immediately", "zeekwritefail.sh"},
+		{"Fail after snapshot", "zeekwritefailslow.sh"},
```

Comment:
Given that the trigger for snapshotting the data is now time based, I would assume just getting rid of the "zeekwritefail" script and going with the timeout test.

Reply:
Also more a matter of tastes but: I find this table based approach more difficult to read as it doesn't follow the convention of the rest of test cases in the file. In my earlier approach I did something similar and @alfred-landrum made a comment that I believe is applicable here: https://github.com/brimsec/zq/pull/399#discussion_r389391872 

I'd rather just create functions `testTaskEndError` and `testEmptySpaceInfo`.

Reply:
I lack strong feelings on these kind of aesthetic points, and would have adjusted if that's implicit convention... but your point about the two tests being redundant is good. I've removed the zeekwritefail script, and we're back to one test!